Config/CUA/flex_config.txt

```diff
@@ -8,6 +8,9 @@ camera.gantryhead.SerialNumberHigh: 0x10FE58
 camera.gantryhead.SerialNumberLow: 0x26A865CD
 camera.gantryhead.FOV-x: 1.095
 camera.gantryhead.FOV-y: 0.824
+light.gantryhead.control: manual
+light.gantryhead.hardware_address: Y.0
```

Comment:
@cfangmeier Even if we don't use this line. It is currently complaining if it is not set

Reply:
Thanks for pointing that out. I'll fix it so `hardware_address` is not needed if the `control` is manual.